amt/services/storage.py

```diff
@@ -57,7 +57,8 @@ def read(self) -> Any:  # noqa
         # todo: this probably has to be moved to 'global scope'
         yaml.add_constructor("!include", yaml_include.Constructor(base_dir=self.base_dir))
         with open(self.path) as f:
-            return yaml.full_load(f)
+            data = yaml.full_load(f)
```

Comment:
Minor: this was a return, but perhaps for debugging you added an extra step? I'd keep the return here.